db_functions.py

```diff
@@ -802,3 +802,13 @@ def update_ad_types(self, ad_type_map):
                                        ad_type_map,
                                        template=insert_template,
                                        page_size=_DEFAULT_PAGE_SIZE)
+
+    def update_ad_last_active_date(self, archive_ids):
+        cursor = self.get_cursor()
+        update_last_active_field_query = (
+                'UPDATE impressions set last_active_date = CURRENT_DATE where archive_id in '
+                'WHERE archive_id = %(archive_id)s')
```

Comment:
is this `where` clause duplicated or am I learning something new about SQL today?

Reply:
you're correct. 

FYI i took over this PR in https://github.com/OnlinePoliticalTransparency/FacebookApiPolAdsCollector/pull/121